corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -55,8 +55,12 @@ <h4 class="panel-title panel-title-nolink">{% trans "Default Search Properties"
           </thead>
           <tbody data-bind="foreach: defaultProperties">
           <tr>
-            <td class="col-sm-4">
+            <td class="col-sm-4" data-bind="css: {'has-error': $parent.commonProperties().indexOf(ko.unwrap(property())) !== -1}">
```

Comment:
This expression is a bit unwieldy, could you extract it as a computed observable with a clear name?

Reply:
True, I have updated it.